core/verkle_witness_test.go

```diff
@@ -364,8 +371,9 @@ func TestProcessVerkleInvalidContractCreation(t *testing.T) {
 			if stemStateDiff.SuffixDiffs[0].NewValue == nil {
 				t.Fatalf("nil new value in BLOCKHASH contract insert")
 			}
-			if *stemStateDiff.SuffixDiffs[0].NewValue != chain[0].Hash() {
-				t.Fatalf("invalid BLOCKHASH value: %x != %x", *stemStateDiff.SuffixDiffs[0].NewValue, chain[0].Hash())
+			if *stemStateDiff.SuffixDiffs[0].NewValue != genesisH {
+				// je sais pas pourquoi, le hash storé est faux. On dirait le empty code hash mais c'est un hasard
```

Comment:
english pls.

Reply:
oops

Reply:
learn some foreign language, you lazy American :wink: